README.md

```diff
@@ -1 +1,26 @@
 # FINDOLOGIC Shopware 6 Plugin Override
+
+This is a basic plugin skeleton which may be used to customize
```

Comment:
Change this to:

...to customize the [FINDOLOGIC Shopware 6 plugin](https://github.com/findologic/plugin-shopware-6).